src/hpc-cache/setup.py

```diff
@@ -35,7 +35,7 @@
 # TODO: Add any additional SDK dependencies here
 DEPENDENCIES = []
 
-with open('README.rst', 'r', encoding='utf-8') as f:
+with open('README.md', 'r', encoding='utf-8') as f:

```

Comment:
seems you don't have README.md file.

Reply:
Because there is this file originally, it is not added by me~
[File link](https://github.com/zhoxing-ms/azure-cli-extensions/blob/hpc_cache_refactor/src/hpc-cache/README.md)